__tests__/resolver.manifest.js

```diff
@@ -6,7 +6,7 @@ const HttpOutgoing = require('../lib/http-outgoing');
 const Manifest = require('../lib/resolver.manifest');
 const Client = require('../');
 const { PodletServer } = require('@podium/test-utils');
-const lolex = require('lolex');
+const lolex = require('@sinonjs/fake-timers');
```

Comment:
Not quite sure about this. What is the reason to switch out lolex here?

Reply:
because the lolex page says it has been deprecated and is now called @sinonjs/fake-timers
<img width="1190" alt="Screenshot 2020-03-05 at 00 04 14" src="https://user-images.githubusercontent.com/1177098/75873518-e508d500-5e74-11ea-90cd-df5959d045cb.png">